src/rez/utils/py23.py

```diff
@@ -38,8 +40,20 @@ def load_module_from_file(name, filepath):
     if six.PY2:
         import imp
         with open(filepath) as f:
-            return imp.load_source(name, filepath, f)
+            module = imp.load_source(name, filepath, f)
+            # Keep the module out of sys.modules. See comment in the `else:`
+            # for more info
+            if name in sys.modules:
+                del sys.modules[name]
```

Comment:
Would it be better to copy the content of `sys.modules` before the import and restore it after the import?

Reply:
Hmm, not sure. I suppose it would guard against the included module adding stuff to `sys.modules`, but I'm not sure what this might break.

Also, this would change the behaviour from the Py3 versions behaviour. Currently if the imported module imports something (say `textwrap`) then this imported module will show up in `sys.modules` for both Py 2 and 3

Reply:
Oh, I didn't know that it behaved like that on python 3. Then forget what I said.